### ISSUE TITLE
VET accessibility - Keyboard users should be able to open dropdown menus

### ISSUE THREAD

**Issue by mhk33** [cul-it/uls#45]
This was open in JIRA. Not sure if it was fixed, but wanted to document it here.

When using keyboard-only navigation, you can open the link to a parent section within the main nav dropdown menus, but you can't open the dropdown menu itself to select links, as mouse users can. Ideally we make the dropdown menu accessible via keyboard.

**Comment by manolobevia** [comment]
CULU theme will take care of this after migrating vet site to the new CULU theme.
